src/coreclr/jit/hwintrinsic.cpp

```diff
@@ -392,6 +428,17 @@ NamedIntrinsic HWIntrinsicInfo::lookupId(Compiler*         comp,
             }
         }
     }
+    else if (isa == InstructionSet_Vector512)
+    {
+        // We support some Vector512 intrinsics when AVX512F, AVX512BW, AVX512CD, AVX512DQ are available.
+        if (!comp->compOpportunisticallyDependsOn(InstructionSet_AVX512F) &&
+            !comp->compOpportunisticallyDependsOn(InstructionSet_AVX512BW) &&
+            !comp->compOpportunisticallyDependsOn(InstructionSet_AVX512CD) &&
```

Comment:
Just noting that at a glance I didn't see anything from `CD` that would make the JIT logic more complicated. It's really just `DetectConflicts` and `LeadingZeroCount`, neither of which are going to be main scenarios for `Vector512<T>` APIs

Reply:
I'm okay with taking out `CD`. I haven't run into any case where it's needed either.

Reply:
Done